pubnub-memberships/src/main/kotlin/com/pubnub/entities/models/consumer/membership/Membership.kt

```diff
@@ -56,13 +76,13 @@ internal fun List<Membership.PartialWithSpace>.toPNChannelWithCustomList(): List
 }
 
 internal fun Membership.PartialWithSpace.toPNChannelWithCustom(): PNChannelMembership.Partial {
-    return PNChannelMembership.Partial(channelId = spaceId, custom = custom, status = status)
+    return PNChannelMembership.Partial(channelId = spaceId.id, custom = custom, status = status)
 }
 
 internal fun List<Membership.PartialWithUser>.toPNUUIDWithCustomList(): List<PNMember.Partial> {
     return map { it.toPNUUIDWithCustom() }
 }
 
 internal fun Membership.PartialWithUser.toPNUUIDWithCustom(): PNMember.Partial {
```

Comment:
What about changing method name from "toPNUUIDWithCustom" to something like toPNMeberPartial  ?